action/action.proto

```diff
@@ -7,24 +7,37 @@ option java_outer_classname = "ActionProto";
 
 service ActionService {
     rpc Arm(ArmRequest) returns(ActionResult) {}
-    rpc TakeOff(TakeOffRequest) returns(ActionResult) {}
+    rpc Disarm(DisarmRequest) returns(ActionResult) {}
+    rpc Takeoff(TakeoffRequest) returns(ActionResult) {}
     rpc Land(LandRequest) returns(ActionResult) {}
+    rpc Kill(KillRequest) returns(ActionResult) {}
+    rpc ReturnToLaunch(ReturnToLaunchRequest) returns(ActionResult) {}
+    rpc TransitionToFixedWings(TransitionToFixedWingsRequest) returns(ActionResult) {}
+    rpc TransitionToMulticopter(TransitionToMulticopterRequest) returns(ActionResult) {}
+    rpc GetTakeoffAltitude(GetTakeoffAltitudeRequest) returns(TakeoffAltitudeResult) {}
+    rpc SetTakeoffAltitude(SetTakeoffAltitudeRequest) returns(ActionResult) {}
+    rpc GetMaximumSpeed(GetMaximumSpeedRequest) returns(MaximumSpeedResult) {}
+    rpc SetMaximumSpeed(SetMaximumSpeedRequest) returns(ActionResult) {}
 }
 
-message UUID {
-    uint64 value = 1;
-}
+message ArmRequest {}
+message DisarmRequest {}
+message TakeoffRequest {}
+message LandRequest {}
+message KillRequest {}
+message ReturnToLaunchRequest {}
+message TransitionToFixedWingsRequest {}
+message TransitionToMulticopterRequest {}
+message GetTakeoffAltitudeRequest {}
 
-message ArmRequest {
-    UUID uuid = 1;
+message SetTakeoffAltitudeRequest {
+    float altitude = 1;
```

Comment:
Units please: `altitude_m`, or `altitudeM`?

Reply:
> "Use CamelCase (with an initial capital) for message names – for example, SongServerRequest. Use underscore_separated_names for field names – for example, song_name. "

From the [guidelines](https://developers.google.com/protocol-buffers/docs/style).

`altitude_m` it is!

Reply:
Ok